Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails'
+gem 'webrick', '~> 1.7'
```

Comment:
yeah this is why you probably have to do the rebase and then force push, to get rid of these changes